package.json

```diff
@@ -13,13 +13,14 @@
     "deploy:anj:ropsten": "node ./bin/index.js minime -n ropsten -c ./data/config/anj.js",
     "deploy:anj:rinkeby": "node ./bin/index.js minime -n rinkeby -c ./data/config/anj.js",
     "deploy:anj:mainnet": "node ./bin/index.js minime -n mainnet -c ./data/config/anj.js",
-    "deploy:court:rpc": "npm run deploy:dai:rpc && npm run deploy:anj:rpc && node ./bin/index.js court -n rpc",
+    "deploy:court:rpc": "rm -f data/output/{minime,court}.rpc.json && npm run deploy:dai:rpc && npm run deploy:anj:rpc && node ./bin/index.js court -n rpc",
     "deploy:court:ropsten": "node ./bin/index.js court -n ropsten",
     "deploy:court:rinkeby": "node ./bin/index.js court -n rinkeby",
-    "deploy:court:mainnet": "node ./bin/index.js court -n mainnet"
+    "deploy:court:mainnet": "node ./bin/index.js court -n mainnet",
+    "postinstall": "cd node_modules/@aragon/court && npm i && cp ../../../truffle-config.js truffle-config.js && npm run prepublishOnly"
   },
   "dependencies": {
-    "@aragon/court": "^1.0.0",
+    "@aragon/court": "aragon/aragon-court#improve_emitted_events",
```

Comment:
Shouldn't we use a version instead?

Reply:
yes, this is wip, I'll replace it with the rc once we have it